### PR TITLE
Some more tests that negative numbers aren't prime

### PR DESCRIPTION
**Pull request by MasterDuke17**
Add some cases that test the MoarVM bug fixed by
https://github.com/MoarVM/MoarVM/pull/1587.

### CODE REVIEW

File: S32-num/is-prime.t [hunk 1]
```diff
@@ -78,6 +78,11 @@ subtest 'Complex.is-prime with Complex that cannot be Real throw' => {
     }
 }
 
-is-deeply is-prime(-2), False, 'negative numbers are *not* prime';
+subtest 'negative numbers are *not* prime' => {
+    plan 3;
+    is-deeply is-prime(-2),  False, 'negative number whose value when interpreted as unsigned is not prime is not prime';
```
Comment (vrurg) [inline S32-num/is-prime.t:83]: `nok` suits better for this kind of test.

**Review by vrurg — CHANGES_REQUESTED** [review]

**Review by vrurg — APPROVED** [review]
